src/module/actor/character/apps/abc-picker/app.svelte

```diff
@@ -13,7 +27,7 @@
     const viewItemSheet: MouseEventHandler<HTMLButtonElement> = async (event): Promise<void> => {
         const uuid = event.currentTarget.closest("li")?.dataset?.uuid ?? "";
         const item = await fromUuid(uuid);
-        item?.sheet.render(true);
+        item?.sheet?.render(true);
```

Comment:
```suggestion
        item?.sheet.render(true);
```

Reply:
As in every case, fix the type rather than dancing around it in runtime unless necessary. `Item#sheet` is never `null`.

Reply:
It is likely because fromUuid is returning a ClientDocument rather than necessarily an item. The alternative is instanceof Item.

Reply:
Ah, I could do a generic and cast it.